charter.html

```diff
@@ -374,6 +374,7 @@ <h2>
         <p>
           At the discretion of the Chairs and subject to a consensus call, the Working Group may designate a Design Team to study a particular subject, and report back to the Working Group their findings and a recommendation. These Design Teams shall consist of a small group of volunteers who commit to investing significant amounts of time on the task over a short, time-bound, period.
         </p>
+        <p>Design teams will operate without putting forward any sensitive information. All teams should operate under the assumption that their data is under the IPR policy and potentially in the open.</p>
```

Comment:
Sorry I don't quite understand the definition of "sensitive information" in the first sentence and it feels like it could be overly broad. Could we cut it in favor of just the second sentence?

Reply:
I found this a little imprecise also.  Maybe the answer here is to say something like:

> Design Teams operate under the same policies as the Working Group.
> For example, the same IPR commitments apply to Design Team contributions as Working Group contributions.

This might more directly address concerns about the potential use of design teams in the working group process.  The point is to select a small set of people to accelerate activity, not to break the process.

I understand that there might be some cases where participants feel it necessary to provide commercially sensitive information as that would have an effect on the conclusions the group makes.  We have had that happen occasionally in similar contexts.  However, that can undermine the transparency of the process in ways we should avoid unless absolutely necessary.  There are tricks we can employ, like Chatham House rule sessions, but they tend to be awkward and very hard to get right.